ios/RN/RNCamera.m

```diff
@@ -310,8 +310,13 @@ - (void)takePicture:(NSDictionary *)options resolve:(RCTPromiseResolveBlock)reso
 {
     AVCaptureConnection *connection = [self.stillImageOutput connectionWithMediaType:AVMediaTypeVideo];
     [connection setVideoOrientation:[RNCameraUtils videoOrientationForDeviceOrientation:[[UIDevice currentDevice] orientation]]];
+    
     [self.stillImageOutput captureStillImageAsynchronouslyFromConnection:connection completionHandler: ^(CMSampleBufferRef imageSampleBuffer, NSError *error) {
         if (imageSampleBuffer && !error) {
+            if ([options[@"autoStopPreview"] boolValue]) {
+                [self stopPreview];
```

Comment:
If you stop preview here, when will you call startPreview again? Shouldn't you call just before resolving the promise?

Reply:
@jgfidelis I thought that could lead to flickering as the resolution of the promise is likely not synchronous with the next JS code to be executed. So it didn’t really solve my use case. Honestly, @aailces has some better ideas in https://github.com/expo/expo/issues/1613#comment-379173558. I think it’d be better to see what she comes up with there and then merge it in here when it lands in `expo`.